discord/guild.py

```diff
@@ -1663,6 +1676,9 @@ async def edit(
         public_updates_channel: TextChannel | None = MISSING,
         premium_progress_bar_enabled: bool = MISSING,
         disable_invites: bool = MISSING,
+        discoverable: bool = MISSING,
```

Comment:
Couldn't test this specific feature flag, but it is editable according to the official docs.

Reply:
I've tested this, it works